docs/.vuepress/config.js

```diff
@@ -0,0 +1,55 @@
+const navbar = require("./navbar");
+const sidebar = require("./sidebar");
+
+module.exports = {
+  base: "/",
+  head: [],
+  locales: {
+    "/": {
+      lang: "en-US",
+      title: "asdf",
```

Comment:
Is this the page title? If so it could probably be longer. e.g. `asdf - Manage multiple runtime versions with a single CLI tool`

Reply:
From the Vuepress docs
>This will be the suffix for all page titles, and displayed in the navbar in the default theme.

This is what the social preview looks like: https://socialsharepreview.com/?url=https://jthegedus.github.io/asdf/

![image](https://user-images.githubusercontent.com/20798510/126607510-28a509ee-1741-4562-8edb-62532a2a7ad1.png)

Vuepress doesn't support further customisation regarding this.